app/Http/Controllers/CalendarController.php

```diff
@@ -0,0 +1,47 @@
+<?php
+
+namespace App\Http\Controllers;
+
+use App\Conference;
+use Calendar;
+
+class CalendarController extends BaseController
+{
+    public function index()
+    {
+        $conferences = Conference::all()->map(function ($conference) {
```

Comment:
at least to start, let's make each of these calls a private method. We may also want to extract to another class or onto the conference class, not sure.